catalog/spatial/registry/registry-common/src/main/java/org/codice/ddf/registry/common/RegistryConstants.java

```diff
@@ -16,6 +16,8 @@
 public class RegistryConstants {
     public static final String REGISTRY_TAG = "registry";
 
+    public static final String REGISTRY_TAG_INTERNAL = "registry.internal";
```

Comment:
These 'internal' metacards are not shared outside the local system. When a remote metacard comes it we store it as an 'internal' metacard but outside people/registries won't see these. I think changing it to registry-remote might make things clearer. Changing it to registry-remote.